lib/timescaledb/schema_dumper.rb

```diff
@@ -112,19 +112,20 @@ def timescale_index_options_for(hypertable)
     def timescale_continuous_aggregates(stream)
       return unless Timescaledb::ContinuousAggregates.table_exists?
 
-      Timescaledb::ContinuousAggregates.all.each do |aggregate|
-        opts = if (refresh_policy = aggregate.jobs.refresh_continuous_aggregate.first)
-                 interval = timescale_interval(refresh_policy.schedule_interval)
-                 end_offset = timescale_interval(refresh_policy.config["end_offset"])
-                 start_offset = timescale_interval(refresh_policy.config["start_offset"])
-                 %Q[, refresh_policies: { start_offset: "#{start_offset}", end_offset: "#{end_offset}", schedule_interval: "#{interval}"}]
-               else
-                 ""
-               end
+      Timescaledb::ContinuousAggregates.all.find_each do |aggregate|
```

Comment:
Thanks for the `find_each` replacement 🧠 


In the future, we may think about also adding a order to keep the caggs name alphabetically or just follow the natural order 🤓